client/config.go

```diff
@@ -175,3 +189,9 @@ func GetConfigFilePath() (string, error) {
 	}
 	return filepath.Join(path, configFileName), nil
 }
+
+func (config *Config) PrintConfigDetails() {
```

Comment:
This probably shouldnt be here - the CLI should handle this, similarly to how remote details are currently printed: https://github.com/ubclaunchpad/inertia/blob/master/remote.go#L121

Reply:
i see ill just remove it